interleave-in-parent.md

```diff
@@ -0,0 +1,172 @@
+---
+title: INTERLEAVE IN PARENT
+summary: Interleaving tables improves query performance by optimizing the key-value structure of closely related table's data.
+toc: false
+toc_not_nested: true
+---
+
+Interleaving tables improves query performance by optimizing the key-value structure of closely related tables, attempting to keep data on the same [key-value range](frequently-asked-questions.html#how-does-cockroachdb-scale) if it's likely to be read and written together.
+
+{{site.data.alerts.callout_info}}Interleaving tables does not affect their behavior within SQL.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## How do interleaved tables work?
```

Comment:
Just a stylistic preference, but in the context of a page like this, I'd prefer something like **How Interleaved Tables Work**.